openstf/model/regressors/custom_regressor.py

```diff
@@ -0,0 +1,69 @@
+# SPDX-FileCopyrightText: 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com> # noqa E501>
+#
+# SPDX-License-Identifier: MPL-2.0
+
+from typing import List, Type
+from abc import abstractmethod
+from importlib import import_module
+import pandas as pd
+import inspect
+from .regressor import OpenstfRegressor
+from ..objective import (
+    RegressorObjective,
+    TEST_FRACTION,
+    VALIDATION_FRACTION,
+    EVAL_METRIC,
+)
+
+
+class CustomOpenstfRegressor(OpenstfRegressor):
+    @staticmethod
+    @abstractmethod
+    def valid_kwargs() -> List[str]:
+        ...
+
+    @property
+    @abstractmethod
+    def objective(self) -> Type[RegressorObjective]:
+        ...
+
+
+def load_custom_model(custom_model_path) -> CustomOpenstfRegressor:
+    path_elts = custom_model_path.split(".")
```

Comment:
What is elts? Maybe we can use longer variable name or add comment to explain? Will make code more readable in long term